projects/plugins/jetpack/modules/videopress/class.videopress-player.php

```diff
@@ -701,9 +701,10 @@ public function html5_dynamic_next() {
 			// phpcs:disable WordPress.WP.EnqueuedResources.NonEnqueuedScript
 			return "<iframe title='" . __( 'VideoPress Video Player', 'jetpack' )
 				. "' aria-label='" . __( 'VideoPress Video Player', 'jetpack' )
-				. "' width='" . esc_attr( $videopress_options['width'] )
+				. "' width='100%"
 				. "' height='" . esc_attr( $videopress_options['height'] )
-				. "' src='" . esc_attr( $iframe_url )
+				. "' style='max-width:" . esc_attr( $videopress_options['width'] )
+				. "px;' src='" . esc_attr( $iframe_url )
```

Comment:
Will also need to set aspect ratio, e.g. I was testing `aspect-ratio: 16 / 9;` but I'm guessing we can calculate it or use the real width/height:

```
aspect-ratio:  $videopress_options['width'] / $videopress_options['height'];
```

Reply:
@lsl were you able to test this last week?  Which method did you use? We've tried following the test instructions in [the comment ](https://github.com/Automattic/jetpack/pull/37025#issuecomment-2072211697) for Simple and Atomic, but the code that gets patched isn't being executed. 

FYI, Feelin' Good is now broken in prod, which is a separate issue I need to look at. The Freddie theme also has the same issue with the Video Press player and can be used instead.